### PR TITLE
Rename from ESA to DMA

### PR DESCRIPTION
**Pull request by esbran**
**This PR fixes**

Renaming from ESA to DMA 
Minor typos

### CODE REVIEW

File: docs/EnterpriseScaleAnalytics-CreateRepository.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data Product Analytics - Create repository from the template
 
-First, you must generate your own respository based off this template respository. To do so, please follow the steps below:
+First, you must generate your own repository based off this template repository. To do so, please follow the steps below:
```
Comment (amanjeetsingh) [inline docs/EnterpriseScaleAnalytics-CreateRepository.md:3]: Change "generate" to "create"

**Review by amanjeetsingh — COMMENTED** [review]
Minor change. 

Clarification - is it "Deploy Data Management & Analytics Scenario" or "Deploy Data Management & Analytics scenario" ? 
Why is the word scenario in uppercase? Please ignore my comment if it has to be in uppercase. Thanks.

**Comment by marvinbuss** [comment]
> Minor change.
> 
> Clarification - is it "Deploy Data Management & Analytics Scenario" or "Deploy Data Management & Analytics scenario" ? Why is the word scenario in uppercase? Please ignore my comment if it has to be in uppercase. Thanks.

Good question. Let's clarify and make sure it is consistent across all repos. 👍

File: docs/EnterpriseScaleAnalytics-CreateRepository.md (hunk 1 above)
Comment (marvinbuss) [reply docs/EnterpriseScaleAnalytics-CreateRepository.md:3]: This must be then changed in all repos and not just this one. This should be consistent.

**Review by marvinbuss — APPROVED** [review]
LGTM!
Will merge after getting the other repos ready.

**Pull request merged**